src/androidTest/java/pl/edu/zut/mad/schedule/login/LoginActivityTest.kt

```diff
@@ -0,0 +1,76 @@
+package pl.edu.zut.mad.schedule.login
+
+import android.content.Intent
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.action.ViewActions.click
+import android.support.test.espresso.action.ViewActions.typeText
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.withId
+import android.support.test.espresso.matcher.ViewMatchers.withText
+import android.support.test.rule.ActivityTestRule
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Rule
+import org.junit.Test
+import pl.edu.zut.mad.schedule.R
+
+internal class LoginActivityTest : LoginActivity() {
```

Comment:
I thought about have this as static class inside test rather than inheritance here :)